src/components/Reports/StaffExpenseReport/DownloadButtonGroup/downloadReport.tsx

```diff
@@ -0,0 +1,74 @@
+import { DateTime } from 'luxon';
+import { Transaction } from 'src/components/Reports/StaffExpenseReport/StaffExpenseReport';
+import { ReportType } from '../Helpers/StaffReportEnum';
+
+const csvHeader = ['Date', 'Category', 'Amount'];
+
+const makeTable = (
+  title: string,
+  transactions: Transaction[],
+  type: ReportType,
+) => {
+  if (type === ReportType.Income) {
+    transactions = transactions.filter((transaction) => transaction.total > 0);
+  } else if (type === ReportType.Expense) {
+    transactions = transactions.filter((transaction) => transaction.total < 0);
+  }
+  const rows = transactions.map((transaction) =>
+    [transaction.month, transaction.category, transaction.total].join(','),
+  );
+  return [title, csvHeader.join(','), ...rows, ''].join('\n');
+};
+
+export const downloadCsv = (
```

Comment:
Thanks for implementing this! I think you'll want to take advantage of `buildURI` to generate the CSV. It will handle escaping values that contain commas more robustly.